orchestrator/workflows/tasks/cleanup_tasks_log.py

```diff
@@ -15,21 +15,20 @@
 from datetime import timedelta
 
 from orchestrator.db import ProcessTable, db
+from orchestrator.settings import app_settings
 from orchestrator.targets import Target
 from orchestrator.types import State
 from orchestrator.utils.datetime import nowtz
 from orchestrator.workflow import ProcessStatus, StepList, done, init, step, workflow
 
 
-@step("Clean up tasks older than one week")
+@step(f"Clean up completed tasks older than {app_settings.DAYS_TO_KEEP_OLD_TASKS} days")
```

Comment:
Dynamische step names kunnen wel gevaarlijk zijn

Reply:
Yeah I was on the fence about this. Changed it to just print the name of the env-var.